README.md

```diff
@@ -1,161 +1,38 @@
-## MvvmCross
-
+<h3 align="center">
+  <img src="docs/img/MvvmCross-logo.png" alt="MvvmCross Logo" />
+</h3>
+
+MvvmCross
+==========
+[![Twitter: @MvvmCross](https://img.shields.io/badge/contact-@MvvmCross-blue.svg?style=flat)](https://twitter.com/MvvmCross)
+![license](https://img.shields.io/github/license/mvvmcross/mvvmcross.svg)
 ![Visual Studio Team services](https://img.shields.io/vso/build/osteost/3472a0ca-2abe-44b8-9897-98bdf5ab78a5/3.svg)
-[![GitHub tag](https://img.shields.io/github/tag/mvvmcross/mvvmcross.svg)](https://github.com/MvvmCross/MvvmCross/releases)
 [![NuGet](https://img.shields.io/nuget/v/MvvmCross.svg)](https://www.nuget.org/packages/MvvmCross/)
 [![NuGet Pre Release](https://img.shields.io/nuget/vpre/MvvmCross.svg)](https://www.nuget.org/packages/MvvmCross/)
+[![GitHub tag](https://img.shields.io/github/tag/mvvmcross/mvvmcross.svg)](https://github.com/MvvmCross/MvvmCross/releases)
 [![MyGet](https://img.shields.io/myget/mvvmcross/v/MvvmCross.svg)](https://www.myget.org/F/mvvmcross/api/v3/index.json)
 [![OpenCollective](https://opencollective.com/mvvmcross/backers/badge.svg)](#backers) 
 [![OpenCollective](https://opencollective.com/mvvmcross/sponsors/badge.svg)](#sponsors)
 [![StackExchange](https://img.shields.io/stackexchange/stackoverflow/t/mvvmcross.svg)](http://stackoverflow.com/questions/tagged/mvvmcross)
 
-MvvmCross is a cross-platform mvvm framework that enables developers to create cross platform apps. 
-
-## Documentation & blog
-
-* See our [official website](http://mvvmcross.com/documentation) for an extensive set of documentation!
-* To stay up-to-date with the latest news go to the [MvvmCross blog](http://mvvmcross.com/blog)!
-
-### Get started
-
-A quick way to get started is to use the Visual Studio MvvmCross [Ninja Coder](https://github.com/asudbury/NinjaCoderForMvvmCross) Extension - download it from [Visual Studio Gallery](https://visualstudiogallery.msdn.microsoft.com/618b51f0-6de8-4f85-95ce-a50c658c7767)
-
-For manual project creation install the necessary packages from [Nuget](https://www.nuget.org/packages?q=mvvmcross)
-
-### Questions & support
-
-* [StackOverflow](http://stackoverflow.com/questions/tagged/mvvmcross)
-* [Xamarin forums](http://forums.xamarin.com)
-* [Slack](https://xamarinchat.herokuapp.com/) join the #mvvmcross channel after you are in
-
-### Documentation and Examples
-
-**The [MvvmCross-Samples](https://github.com/MvvmCross/MvvmCross-Samples) repo contains the latest samples. See the [MvvmCross Wiki](https://github.com/MvvmCross/MvvmCross/wiki) for additional articles and information.**
+👀 Check out [mvvmcross.com](https://www.mvvmcross.com) to get started with MvvmCross 👀
 
-**Samples and Videos:**
+MvvmCross is a cross-platform MVVM framework. It enables developers to create apps using the MVVM pattern on Xamarin and Windows (UWP & WPF). This allows for better code sharing by allowing you to share behavior and business logic between platforms.
```

Comment:
Can we list here the same platforms the docs enumerates?
Xamarin.iOS, Xamarin.Android, Xamarin.Mac, Xamarin.Forms, Universal Windows Platform (UWP) and Windows Presentation Framework (WPF)